pkg/execute/action.go

```diff
@@ -0,0 +1,88 @@
+package execute
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"text/tabwriter"
+
+	"github.com/kubeshop/botkube/pkg/config"
+)
+
+func (e *DefaultExecutor) runActionCommand(ctx context.Context, args []string, clusterName string) (string, error) {
+	if len(args) < 2 {
+		return "", errInvalidCommand
+	}
+
+	validSubCmd := false
+	for _, cmdName := range []string{"action", "actions", "act"} {
+		if args[1] == cmdName {
+			validSubCmd = true
+			break
+		}
+	}
+	if !validSubCmd {
+		return fmt.Sprintf("'%s' is not a valid subcommand", args[1]), nil
+	}
+
+	var cmdVerb = args[0]
+	defer func() {
+		cmdToReport := fmt.Sprintf("%s %s", args[1], cmdVerb)
+		e.reportCommand(cmdToReport, false)
+	}()
+
+	actions, err := e.cfgManager.ListActions(ctx)
+	if err != nil {
+		return "Failed to list actions", err
+	}
```

Comment:
Actually a dedicated call to get ConfigMap data is not needed - we can read the actions directly from the config. We reload Botkube anytime the config changes, so we can assume it is always up to date (as it is the one loaded into the Botkube app) 👍 

This is the same as the edit executor: https://github.com/kubeshop/botkube/blob/main/pkg/execute/edit.go - it also takes a config, processes it and based on it it displays a modal with all sources configured.